src/chainparams.cpp

```diff
@@ -78,7 +78,7 @@ class CMainParams : public CChainParams {
         consensus.nPowTargetSpacing = 10 * 60;
         consensus.fPowAllowMinDifficultyBlocks = false;
         consensus.fPowNoRetargeting = false;
-        consensus.nRuleChangeActivationThreshold = 1916; // 95% of 2016
+        consensus.nRuleChangeActivationThreshold = 1815; // 90% of 2016
```

Comment:
commit ffe33dfbd4c3b11e3475b022b6c1dd077613de79: While touching this line, maybe also add a link somewhere to the section "Modified thresholds" in https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki#support-for-future-changes ?